source/docs/software/roborio-info/roborio2-imaging.rst

```diff
@@ -0,0 +1,56 @@
+RoboRIO2 Imaging
+================
+
+The NI roboRIO 2.0 boots from a microSD card configured with an appropriate boot image containing the NI Linux Real-Time OS, drivers, and libraries specific to FRC. There are two ways to create the SD card -- with the roboRIO :ref:`using the imaging tool <docs/zero-to-robot/step-3/imaging-your-roborio:Imaging your roboRIO>`, or with a laptop and an SD burner application.
+
+microSD Requirements
+--------------------
+
+The NI roboRIO 2.0 controller supports microSD cards.
+
+Operation Tips
+--------------
+
+The NI roboRIO 2.0 controller requires a fully inserted SD card containing a valid image in order to boot and operate as intended.
+
+If the SD card is removed while powered, the controller will hang. Once the SD card is replaced, the controller will need to be restarted using the reset button, or be power cycled.
+
+No damage will result from SD card removal or insertion while powered, but best practice is to perform these operations while unpowered.
+
+Imaging Using a Computer
```

Comment:
How else would we image...? Aliens?

Reply:
Did you have a suggestion that comment doesn't seem helpful.

Maybe: "Imaging Using a USB Cable to the RIO with the microSD Card Inserted" but that seems really long so I thought that imaging using a computer got the point across.

Reply:
Just "Imaging the SD Card"

Reply:
I want to convey that there are 2 options for imaging and this is just the direct imaging.